.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,10 +17,12 @@ Closes # <-- _insert Issue number if one exists_
 
 ## Checklist
 
+- [ ] formatted title correctly? (_see [here](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CONTRIBUTING.md#submit-a-pull-request)
+      for formatting details_)
 - [ ] added appropriate tests?
 - [ ] performed checkstyle check locally?
 - [ ] added/updated copyright headers?
 - [ ] documented public classes/methods?
 - [ ] added/updated relevant documentation?
-- [ ] added relevant details to the changelog? (skip with label `no-changelog`)
-- [ ] linked GitHub project? (see the section on the right-hand side -->)
+- [ ] added relevant details to the changelog? (_skip with label `no-changelog`_)
+- [ ] linked GitHub project? (_see the section on the right-hand side -->_)
```

Comment:
can non-committers do that? also, does that not happen automatically?

Reply:
No, they can't. And same as for issues, as described [here](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/854#discussion_r822608819). The initial assignment could be easily automated if we had one project that was mandatory. But people were raising the eventuality of having multiple ones in GitHub's project tab. So we need to come up with an automatism based on labels or pr content or something like this. As a pr can only be merged by a committer, this can be done manually for now.